packages/wdio-cli/src/index.js

```diff
@@ -25,16 +22,21 @@ export const run = async () => {
     }
 
     /**
-     * if yargs doesn't run a command from the command directory
-     * we verify if a WDIO config file exists in the given "path" parameter
-     * if so, we assume the user ran `wdio path/to/wdio.conf.js` and we execute `wdio run` command
+     * The only way we reach this point is if the user runs the binary without a command (i.e. wdio wdio.conf.js)
+     * We can safely assume that if this is the case, the user is trying to execute the `run` command as it
+     * was previous to https://github.com/webdriverio/webdriverio/pull/4402
+     *
+     * Since the `run` command verifies if the configuration file exists before executing
+     * we don't have to check that again here.
```

Comment:
@christian-bromann Do you think it'll make sense to verify if the config file exists before executing yargs? This check would be ignored if the user runs the config command.

Reply:
what is the expected result if config exists and user runs wdio config?
Should we ask to overwrite config or something?
(sorry if I misunderstood the question)

Reply:
@mgrybyk  If you run `wdio config` and there is a config already then you will overwrite it. That's the behaviour since a while if I recall correctly. We could add a check to see if a wdio.conf.js file exists and either exit or ask to overwrite.

My question was if we should verify if the config file exists before running a command other than `config` instead of verifying it inside each individual command. Currently we do this check in the [run](https://github.com/webdriverio/webdriverio/blob/master/packages/wdio-cli/src/commands/run.js#L117) and [install](https://github.com/webdriverio/webdriverio/blob/master/packages/wdio-cli/src/commands/install.js#L60) commands.